estate/models/estate_property.py

```diff
@@ -0,0 +1,156 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api, _ #importing _ for the text of the error(unsure).
+from odoo.exceptions import UserError, ValidationError #importing to throw error in ui view
+from dateutil.relativedelta import relativedelta # to do operation on datetime class.
+import odoo.tools.float_utils as floatUtil
```

Comment:
```suggestion
from dateutil.relativedelta import relativedelta # to do operation on datetime class.

from odoo import _, api, fields, models #importing _ for the text of the error(unsure).
from odoo.exceptions import UserError, ValidationError #importing to throw error in ui view
import odoo.tools.float_utils as floatUtil
```

Generally we first make external imports and then we import from odoo in alphabetical order.
Need to adapt in all files